lm_eval/models/optimum_lm.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+from lm_eval.api.registry import register_model
+from lm_eval.models.huggingface import HFLM
+
+
+@register_model("optimum-causal")
+class OptimumLM(HFLM):
+    """
+    Optimum Intel provides a simple interface to optimize Transformer models and convert them to \
+    OpenVINO™ Intermediate Representation (IR) format to accelerate end-to-end pipelines on \
+    Intel® architectures using OpenVINO™ runtime.
+    """
+
+    def __init__(
+        self,
+        device = "cpu",
+        **kwargs,
+    ) -> None:
+        
+        if "backend" in kwargs:
+            # optimum currently only supports causal models.
+            assert kwargs["backend"] == "causal"
+        else:
+            raise Exception("Please be sure your model is a `causal` model.")
+
+        assert device == "cpu"
+        super().__init__(device=device, **kwargs)
```

Comment:
```suggestion
        super().__init__(
            device=device,
            backend=kwargs.get("backend", "causal"),
            **kwargs,
        )
```

Reply:
Thanks for your message. Yes, I have run it on my own and results / runtimes look good.

Reply:
Great! Could you accept the suggestion here overriding backend?

Reply:
Added your suggestion as well as Alexander's. For the readme file please consider the part I added for 'Openvino' and ignore the rest of changes. Thanks!